core/validation/utils.py

```diff
@@ -12,3 +17,95 @@ def remove_duplicate_indexes(df: pd.DataFrame):
     :return: the DataFrame without duplicate indexes
     """
     return df[~df.index.duplicated(keep="first")]
+
+
+def get_start_date_column_for_failures(failed_rows: pd.DataFrame) -> list[tuple] | None:
+    """
+    Retrieves the 'Start_Date' column values for the specified indexes.
+
+    :param failed_rows: A pandas DataFrame containing the data of rows that have failed validation.
+    :return: A list of tuples, where each tuple contains an index and the corresponding 'Start_Date' value.
+    """
+    start_dates_series = failed_rows["Start_Date"] if "Start_Date" in failed_rows.columns else None
+    if start_dates_series is not None:
+        test = [(index, str(start_date)) for index, start_date in start_dates_series.iteritems()]
+        return test
+
+
+def get_cell_indexes_for_outcomes(failed_rows: pd.DataFrame) -> str:
```

Comment:
If we then return series instead then these functions need to be modified to expect a series